website_sale_one_step_checkout/README.rst

```diff
@@ -0,0 +1,41 @@
+One Step Checkout
+=================
+
+Provides an All-In-One Checkout for Your Odoo Customer
+
+.. image:: https://github.com/blooparksystems/website/blob/10.0-wip-website_sale_osc/website_sale_one_step_checkout/static/description/osc.png
```

Comment:
Please use relative paths for all images here, e. g. "/website_sale_one_step_checkout/static/description/osc.png". In addition create new screenshots, 2880x1800px is to big. The screen should have a maximum of 826px to not break the view layout later in odoo.